Samples/Notifications/App/README.md

```diff
@@ -54,6 +54,13 @@ At the very least, that means adding an entry to describe your scenario, near th
 Verify your new scenario builds (without warnings), that it works as expected and has been implemented for all flavors of the sample (C++/C#, Packaged/Unpackaged, etc.).
 Go to the [WindowsAppSDK-Samples](https://github.com/microsoft/WindowsAppSDK-Samples) GitHub and follow the Contributing Guidelines (at the bottom of the readme) and submit ytour code changes for peer review.
 
+6. **Converting the Unpackaged Sample to Packaged Sample** -
+Sample can be converted to Packaged by following the guidance: https://docs.microsoft.com/en-us/windows/apps/windows-app-sdk/single-project-msix?tabs=cpp
+   - **For a C++ Sample** - Set the below properties:
+         1. Open the vcxproj and set the AppxPackage property to true and WindowsPackageType to MSIX
+         3. Copy the provided sample Package.appxmanifest file under App/Resources/ to the unpackaged app solution. Note the provided manifest is only for reference modify the parameters as per the documentation:
+            https://docs.microsoft.com/en-us/windows/apps/windows-app-sdk/notifications/app-notifications/app-notifications-quickstart?tabs=cs
+
```

Comment:
After copying the package manifest, users will have to add it to their project in visualstudio (add existing item))

Reply:
and got this error when building the sample:
1>D:\Program Files\Microsoft Visual Studio\2022\Enterprise\MSBuild\Current\Bin\amd64\Microsoft.Common.CurrentVersion.targets(823,5): error : The BaseOutputPath/OutputPath property is not set for project 'CppUnpackagedAppNotifications.vcxproj'.  Please check to make sure that you have specified a valid combination of Configuration and Platform for this project.  Configuration='Debug'  Platform='x64'.  This error may also appear if some other project is trying to follow a project-to-project reference to this project, this project has been unloaded or is not included in the solution, and the referencing project does not build using the same or an equivalent Configuration or Platform.